config/robots.rb

```diff
@@ -1870,4 +1870,100 @@
                    target_state: 'passed'
                  }
                })
+
+  custom_robot('bravo-lthr-384-rt-to-lthr-384-pcr-1-and-2',
+               name: 'bravo LTHR-384 RT => LTHR-384 PCR 1 and 2',
+               beds: {
+                 bed(9).barcode => {
+                   purpose: 'LTHR-384 RT',
+                   states: ['passed'],
+                   label: 'Bed 9'
+                 },
+                 bed(4).barcode => {
+                   purpose: 'LTHR-384 PCR 1',
+                   states: ['pending'],
+                   label: 'Bed 4',
+                   target_state: 'passed',
+                   parent: bed(9).barcode
+                 },
+                 bed(6).barcode => {
+                   purpose: 'LTHR-384 PCR 2',
+                   states: ['pending'],
+                   label: 'Bed 6',
+                   target_state: 'passed',
+                   parent: bed(9).barcode
+                 }
+               })
+
+  custom_robot('mosquito-lthr-384-pcr-1-and-2-to-lthr-384-lib-pcr-1-and-2',
+               name: 'Mosquito LV LTHR-384 PCR 1 and 2 => LTHR-384 Lib PCR 1 and 2',
+               beds: {
+                 bed(1).barcode => {
+                   purpose: 'LTHR-384 PCR 1',
+                   states: ['passed'],
+                   label: 'Bed 1'
+                 },
+                 bed(3).barcode => {
+                   purpose: 'LTHR-384 PCR 2',
+                   states: ['passed'],
+                   label: 'Bed 3'
+                 },
+                 bed(2).barcode => {
+                   purpose: 'LTHR-384 Lib PCR 1',
+                   states: ['pending'],
+                   label: 'Bed 2',
+                   target_state: 'passed',
+                   parent: bed(1).barcode
+                 },
+                 bed(4).barcode => {
+                   purpose: 'LTHR-384 Lib PCR 2',
+                   states: ['pending'],
+                   label: 'Bed 4',
+                   target_state: 'passed',
+                   parent: bed(3).barcode
+                 }
+               })
+
+  custom_robot('bravo-lhr-384-pcr-1-and-2-to-lhr-384-cdna',
```

Comment:
Don't think this robot should be here? Looks like copy-pasted from the existing mosquito version, and it morphed into the one below?